packages/vapor/scss/components/tile.scss

```diff
@@ -0,0 +1,65 @@
+.tile {
+    position: relative;
+    top: 20px;
+    left: 20px;
+    width: 264px;
+    height: auto;
```

Comment:
What is this one for?

Reply:
the height? :eyes: 
I can try and remove it to see if nothing breaks :+1: